test/testdata/resolver/recover_from_bad_superclass.rb

```diff
@@ -0,0 +1,37 @@
+# typed: true
```

Comment:
can you add an symbol-table.exp file for this? I'd like to have some `.exp` file mentioning `StubSuperClass`